config.json

```diff
@@ -820,6 +820,14 @@
         "prerequisites": [],
         "difficulty": 4
       },
+      {
+        "slug": "word-count",
```

Comment:
You already added `word-count`, see https://github.com/exercism/cairo/pull/304/files
Delete these files from the PR.

Reply:
Apologies for this error. I had issues which prompted this error along the line. I'd delete it from the PR.

Reply:
I have deleted these files from the PR. Please, let me know if there are other changes to make.

Reply:
No problem at all, it happens!

Reply:
The files are still in the PR :sweat_smile: